specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -203,6 +204,7 @@ const processFieldType: {
       step: f.parseFloat(getProperty('step')),
       minLength: f.parseInt(getProperty('minLength')),
       maxLength: f.parseInt(getProperty('maxLength')),
+      isBold: getProperty('isBold')?.toLowerCase() === 'true',
```

Comment:
why do you read `isBold` here if you already read `isBold` on the Cell level in https://github.com/specify/specify7/blob/issue-3095/specifyweb/frontend/js_src/lib/components/FormParse/cells.ts#L359-L361